src/Alterway/Bundle/RestProblemBundle/Resources/config/services.yml

```diff
@@ -12,5 +12,7 @@ services:
   aw.rest_exception.listener:
     class: %aw.rest_exception.listener.class%
     arguments: [%kernel.debug%]
+    calls:
+      - [setLogger, ['@?logger']]
```

Comment:
If there is no logger defined in the service container, the method is not even called.